docs/getting-started.mdx

```diff
@@ -8,46 +8,59 @@ import { AssetsProvider } from '@site/src/components/assets-context';
 import DownloadCard from '@site/src/components/download-card';
 import DownloadCardStyles from '@site/src/css/download-card.module.css';
 
-
-## Get an Account
-
-<Columns className='text--center getting-started-cards'> 
-  <Column style={{paddingBottom: 20}}>
-    <Card shadow='tl'>
-      <CardHeader>
-        <h3>Hosted zrokNet</h3>
-      </CardHeader>
-      <CardBody>
-        Use NetFoundry's public zrok instance.
-      </CardBody>
-      <CardFooter>
-        <a href="https://myzrok.io/">
-        <button className='button button--secondary button--block'>Get an Account</button>
-        </a>
-      </CardFooter>
-    </Card>
-  </Column>
-  <Column style={{paddingBottom: 20}}>
-    <Card shadow='tl'>
-      <CardHeader>
-        <h3>Self-Hosted zrok</h3>
-      </CardHeader>
-      <CardBody>
-        Run a zrok instance on Linux, Docker, or Kubernetes.
-      </CardBody>
-      <CardFooter>
-        <a href="/docs/category/self-hosting/">
-        <button className='button button--secondary button--block'>Guides</button>
-        </a>
-      </CardFooter>
-    </Card>
-  </Column>
+## Secure resource sharing made easy
+
+<Columns className='text--center getting-started-cards'>
+    <Column style={{paddingBottom: 20}}>
+        <Card shadow='tl'>
+            <CardHeader>
+                <h3>Hosted zrok</h3>
+            </CardHeader>
+            <CardBody>
+                Use NetFoundry's public zrok instance.
+            </CardBody>
+            <CardFooter>
+                <a href="https://myzrok.io/">
+                    <button className='button button--secondary button--block'>Get an Account</button>
+                </a>
+            </CardFooter>
+        </Card>
+    </Column>
+    <Column style={{paddingBottom: 20}}>
+        <Card shadow='tl'>
+            <CardHeader>
+                <h3>Self-Hosted zrok</h3>
+            </CardHeader>
+            <CardBody>
+                Run a zrok instance on Linux, Docker, or Kubernetes.
+            </CardBody>
+            <CardFooter>
+                <a href="/docs/category/self-hosting/">
+                    <button className='button button--secondary button--block'>Guides</button>
+                </a>
+            </CardFooter>
+        </Card>
+    </Column>
 </Columns>
 
+* Sign up for a hosted zrok account in the link above to enable your account token
+* Download the zrok binary for your platform [here](#installing-the-zrok-command)
+* Run `zrok enable <your_account_token>` to enable your shell environment for sharing
+* Run `zrok share public localhost:8080` to start your first share!
```

Comment:
```suggestion
* Run `zrok share public 8080` to start your first share!
```

The default target address is `127.0.0.1`, so we could shorten it like this if you want to minimize it visually.

Reply:
i prefer the shorter version as it looks shorter and easier. i'd rather see it without the localhost nor with 127.0.0.1, personally

Reply:
The inferred 127.0.0.1 is a not-so-obvious feature. If someone wants to terminate on something other than localhost, now they already now how to do that.

Reply:
people who want that know how to read the doc and know how to read the help. people who don't know this will be lost regardless. i'd prefer to keep it looking as small and sleek as possible

Reply:
I agree with Clint on keeping it as small/sleek as possible